### PR TITLE
Delete switch properties when switch is deleted

### PR DESCRIPTION
**Pull request by rozdy**
This change is [Reviewable](https://reviewable.io/reviews/telstra/open-kilda/2836)

### CODE REVIEW

File: services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java [hunk 1]
```diff
@@ -151,6 +151,8 @@ public boolean deleteSwitch(SwitchId switchId, boolean force) throws SwitchNotFo
             // forceDelete() removes switch along with all relationships.
             switchRepository.forceDelete(sw.getSwitchId());
         } else {
+            switchPropertiesRepository.findBySwitchId(sw.getSwitchId())
+                    .ifPresent(sp -> switchPropertiesRepository.delete(sp));
             // delete() is used to be sure that we wouldn't delete switch if it has even one relationship.
```
Comment (niksv) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java]: Maybe we need to delete properties in `force` case too?

Comment (rozdy) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java]: Fixed

File: services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java [hunk 2]
```diff
@@ -147,6 +147,8 @@ public boolean deleteSwitch(SwitchId switchId, boolean force) throws SwitchNotFo
         Switch sw = switchRepository.findById(switchId)
                 .orElseThrow(() -> new SwitchNotFoundException(switchId));
 
+        switchPropertiesRepository.findBySwitchId(sw.getSwitchId())
```
Comment (timofei-durakov) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java]: why is it not in a transaction?

Comment (rozdy) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java]: Added transaction

**Pull request merged**